src/pages/Extraction.tsx

```diff
@@ -46,13 +48,34 @@ function buildExtractionTableData(ctx: ExtractionContext) {
     .flat();
 }
 
-function Extraction() {
-  const [current, send] = useMachine(() => extractionMachine.withContext({ labwares: [], workNumber: '' }));
+/***
+ * Equipments with 'extract' category
+ */
+type ExtractionEquipments = {
+  equipments: EquipmentFieldsFragment[];
+};
+
+const extractionEquipments = (equipments: EquipmentFieldsFragment[]) => {
+  const methods: Array<{ value: number; label: string }> = [];
+
+  equipments.forEach((equipment) => {
+    methods.push({ value: equipment.id, label: `Automated - ${equipment.name}` });
+  });
+  methods.push({ value: -1, label: 'Manual' });
```

Comment:
Why the value is -1? While sending request equipmentId<0 is considered as undefined

Reply:
the 'value' can only hold a number or string, as it is assigned to 'OptionTemplate' type. as the method is manual, no equipment id should be assigned.

Reply:
Ok, got it, that makes sense.  👍